.github/workflows/release.yml

```diff
@@ -194,6 +194,15 @@ jobs:
       - uses: actions/checkout@v2
         if: env.VERSION_BEFORE != env.VERSION_AFTER
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: arm64,amd64
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@master
+
       - name: Upload build to Docker Hub
         if: env.VERSION_BEFORE != env.VERSION_AFTER
         uses: docker/build-push-action@v1
```

Comment:
i think this is the change we'll have to make: `docker/build-push-action@v1` -> `docker/build-push-action@v2`